python_orocos_kdl/PyKDL/std_string.sip

```diff
@@ -39,7 +39,7 @@
     // If argument is a Python string, assume it's UTF-8
      if (sipIsErr == NULL)
 #if PY_MAJOR_VERSION < 3
-        return (PyString_Check(sipPy) || PyUnicode_Check(sipPy));
+        return (PyBytes_Check(sipPy) || PyUnicode_Check(sipPy));
```

Comment:
It looks like the `PyBytes_X` functions are Python3 only (https://docs.python.org/2.7/c-api/string.html) so the `PyString_X` calls within the `#if PY_MAJOR_VERSION < 3` should stay unchanged AFAICT

Reply:
Seems right. As stated in the PR, changes within the SIPs are by @rkojcev, but if needed I can rebase and remove those changes.